src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -43,13 +43,17 @@ const CustomFieldInputSingleChoice = forwardRef(function CustomFieldInputSingleC
   const clear = () => {
     setValue(undefined);
     setSearchValue(undefined);
+    inputRef.current.focus();
   };
 
   const clearIcon = () => {
     if (!props.readOnly && (value || searchValue)) {
       return (<Icon
         name={iconClear.id}
         onClick={clear}
+        onEnter={clear}
```

Comment:
We can clean up the interface by using more user-centric names: instead of `onClick` and `onEnter` we can say `onPress`.

Reply:
Though, to stay backwards compatible, we might want to avoid deprecating `onClick` (yet). (Or we can backfill).